ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -403,15 +403,21 @@ In particular, the API method `eth_getBlockReceipts` MUST return the receipts co
 #### Named blocks
 
 The Ethereum Mainnet APIs allow for retrieving blocks by named parameters that the API server resolves based on their consensus mechanism.
-Other than the _earliest_ (genesis) named block, which MUST be interpreted in the same manner, all other named blocks are mapped to SAE in terms of the _execution_ status of blocks and MUST be interpreted as follows:
+C-Chain does not currently support _pending_, _safe_, and _finalized_ labels.
```

Comment:
I don't think this comment is correct. The C-chain aliases `pending`, `latest`, `safe`, and `finalized` as the last accepted block.

Reply:
Fixed.